package.json

```diff
@@ -4,6 +4,7 @@
   "description": "A utility to build lh3.googleusercontent URLs",
   "main": "index.js",
   "type": "module",
+  "types": "index.d.ts",
```

Comment:
You'll probably also need to add this file to "files"

Reply:
I didn't see a `files` property, so I've added one now in a1d5154eff961fc5d34a36ba7c9c06c0979ff2aa.